cmd/multi-factor-auth/swagger/data.go

```diff
@@ -223,3 +223,52 @@ type _ struct {
 	// required:true
 	Payload requests.SignMultipleTransactions
 }
+
+// swagger:route POST /set-security-mode-no-expire Guardian setSecurityModeNoExpireRequest
```

Comment:
```suggestion
// swagger:route POST /set-security-mode-no-expire Guardian securityModeNoExpireRequest
```

this way swagger knows what parameters to expect for this request, binding them to the already defined structure securityModeNoExpireRequest

Reply:
updated